Ensure all release-1.y-blocking jobs have a sigOwner

spiffxp opened the pull request:
Next PR for kubernetes/test-infra#5303, followed pattern set by kubernetes/test-infra#5306

Broke up commits by sig that was assigned

I took one extra step for this, where I assigned sig-cluster-lifecycle to pretty much any job that was upgrade or downgrade related.  Justification: the `[Feature:...]` tags used were in `test/e2e/lifecycle`, or I followed a pattern of other upgrade tests already referring to sig-cluster-lifecycle.

This hopefully catches more than just release-foo-blocking, but also some or most of the release-foo-upgrade dashboards

I could see a case being made that the GKE jobs should all be bulk assigned to sig-gcp, upgrade or not.

Job assignments break down as follows
```
ci-kubernetes-e2e-gce-1-6-1-7-upgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-1-6-1-7-upgrade-cluster-new: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-1-6-1-7-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-alpha-features-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-alpha-features-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-gci-latest-upgrade-etcd: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-master-new-downgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-master-new-downgrade-cluster-parallel: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-new-master-upgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-new-master-upgrade-cluster-new: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-new-master-upgrade-cluster-parallel: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-new-master-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-reboot-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-reboot-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-scalability-release-1-7: UNKNOWN -> sig-scale
ci-kubernetes-e2e-gce-serial-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-serial-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-slow-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-slow-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gce-stable1-stable2-downgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gce-stable1-stable2-downgrade-cluster-parallel: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gci-gce-alpha-features-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-alpha-features-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-alpha-features-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-reboot-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-reboot-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-reboot-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-scalability-release-1-7: UNKNOWN -> sig-scale
ci-kubernetes-e2e-gci-gce-scalability-stable1: UNKNOWN -> sig-scale
ci-kubernetes-e2e-gci-gce-serial-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-serial-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-serial-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-slow-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-slow-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-slow-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gce-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-alpha-features: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-flaky: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-ingress-release-1-6: UNKNOWN -> sig-network
ci-kubernetes-e2e-gci-gke-reboot-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-reboot-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-reboot-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-serial-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-serial-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-serial-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-slow-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-slow-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-slow-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gci-gke-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-alpha-features-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-alpha-features-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-cvm-1-6-gci-1-7-upgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-cvm-1-6-gci-1-7-upgrade-cluster-new: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-cvm-1-6-gci-1-7-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-cvm-1-6-gci-1-8-upgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-cvm-1-6-gci-1-8-upgrade-cluster-new: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-cvm-1-6-gci-1-8-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-1-6-gci-1-7-upgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-1-6-gci-1-7-upgrade-cluster-new: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-1-6-gci-1-7-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-master-gci-new-downgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-master-gci-new-downgrade-cluster-parallel: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-new-gci-master-upgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-new-gci-master-upgrade-cluster-new: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-new-gci-master-upgrade-cluster-parallel: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-new-gci-master-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-stable1-gci-stable2-downgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-gci-stable1-gci-stable2-downgrade-cluster-parallel: UNKNOWN -> sig-cluster-lifec
ci-kubernetes-e2e-gke-gci-stable3-gci-stable1-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-reboot-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-reboot-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-serial-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-serial-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-slow-release-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-slow-release-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-e2e-gke-staging-default-latest-upgrade-cluster: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-staging-default-latest-upgrade-cluster-new: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-e2e-gke-staging-default-latest-upgrade-master: UNKNOWN -> sig-cluster-lifecycle
ci-kubernetes-soak-gce-1-6: UNKNOWN -> sig-gcp
ci-kubernetes-soak-gce-1-7: UNKNOWN -> sig-gcp
ci-kubernetes-soak-gci-gce-stable1: UNKNOWN -> sig-gcp
ci-kubernetes-soak-gci-gce-stable2: UNKNOWN -> sig-gcp
ci-kubernetes-soak-gci-gce-stable3: UNKNOWN -> sig-gcp
ci-kubernetes-soak-gke-gci: UNKNOWN -> sig-gcp
ci-kubernetes-test-go-release-1.6: UNKNOWN -> sig-testing
ci-kubernetes-test-go-release-1.7: UNKNOWN -> sig-testing
ci-kubernetes-test-go-release-1.8: UNKNOWN -> sig-testing
ci-kubernetes-verify-release-1.6: UNKNOWN -> sig-testing
ci-kubernetes-verify-release-1.7: UNKNOWN -> sig-testing
ci-kubernetes-verify-release-1.8: UNKNOWN -> sig-testing
```

BenTheElder commented:
> iEnsure all release-1.y-blocking jobs have a sigOwner

Sponsored by Apple? :joy:

spiffxp commented:
Sponsored by vim 😆

spiffxp commented:
FYI @kubernetes/sig-scalability-test-failures @kubernetes/sig-testing-test-failures @kubernetes/sig-cluster-lifecycle-test-failures @kubernetes/sig-gcp-test-failures 

FYI @kubernetes/sig-release-members

BenTheElder commented:
/test all

porridge commented:
Why sig-scale and not sig-scalability?

krzyzacy commented:
besides comment from @porridge 
/lgtm
/hold

shyamjvs commented on jobs/config.json:
```diff
@@ -2586,7 +2586,7 @@
     "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "UNKNOWN"
+      "sig-scale"
```
s/sig-scale/sig-scalability/
Renamed for other jobs in https://github.com/kubernetes/test-infra/pull/5359.

spiffxp replied:
thanks, I didn't want to slime that into this PR, I'll rebase

spiffxp commented:
/hold cancel
renamed as suggested (and updated https://github.com/kubernetes/community/issues/703, we probably have some other discrepancies if someone cares to correct)
